lib/utils/fs/dirExists.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const fse = require('./fse');
+
+function dirExists(path) {
+  return fse
+    .lstatAsync(path)
+    .then(stats => stats.isDirectory())
+    .catch(() => false);
```

Comment:
Should be a second callback to `then`.

Also ideally should `return false` only on `ENOENT` (other errors signal permission and other FS issues, which should be exposed)